terraform/api_gateway_domain_name.tf.json.template.py

```diff
@@ -72,7 +72,23 @@
                         "evaluate_target_health": True,
                     }
                 }
-            }
-        } for lambda_name, gateway_id in gateway_ids.items() if gateway_id
+            },
+        },
+         **({
+             "aws_cloudwatch_log_group": {
+                 lambda_name: {
+                     "name": "/aws/apigateway/azul-%s-%s" % (lambda_name, config.deployment_stage),
```

Comment:
> I would rather inline the contents of `log_api_gateway.py` but I don't think that's possible.
 It could be inlined with a couple calls to the awscli. I didn't realize (until just now) that it's a dependency so I thought it better to use Boto since I know it's installed. If you'd rather me refactor with the cli, let me know.

> The only other issue I have is spaces and other shell meta characters in the interpolated variables.

Using the awscli won't work around this, but I suppose I could encase the interpolated arguments in single quotes. Practically speaking though I don't think it would be an issue.